cni/network/network.go

```diff
@@ -653,9 +642,16 @@ func (plugin *NetPlugin) createNetworkInternal(
 		ServiceCidrs:                  ipamAddConfig.nwCfg.ServiceCidrs,
 	}
 
-	setNetworkOptions(ipamAddResult.ncResponse, &nwInfo)
+	// check if ipamMode is dualStackOverlay mode, then set IPv6Mode to dualStackOverlay mode
+	if ipamAddConfig.nwCfg.IPAM.Mode == string(util.DualStackOverlay) {
+		nwInfo.IPV6Mode = ipamAddConfig.nwCfg.IPAM.Mode
+	}
```

Comment:
i dont think we need this check. directly assign it without if check

Reply:
We set IPV6Mode before: ipamAddConfig.nwCfg.IPV6Mode, so ipv6Mode was set before and only if it's dualStackOverlay mode, we change ipv6Mode to dualStackOverlay